src/ekobox/cards.py

```diff
@@ -0,0 +1,109 @@
+"""Tools to generate runcards."""
+import os
+from math import nan
+from typing import Any, Dict
+
+import numpy as np
+import yaml
+
+from eko import basis_rotation as br
+from eko.io import runcards
+
+Card = Dict[str, Any]
+
+_theory = dict(
+    order=[1, 0],
+    couplings=dict(alphas=[0.118, 91.2], alphaem=[0.007496252, nan]),
+    num_flavs_ref=None,
+    num_flavs_init=None,
+    num_flavs_max_as=6,
+    num_flavs_max_pdf=6,
+    intrinsic_flavors=[4],
+    quark_masses={q: [mq, nan] for mq, q in zip((2.0, 4.5, 173.07), "cbt")},
+    quark_masses_scheme="POLE",
+    matching=[1.0, 1.0, 1.0],
+    fact_to_ren=1.0,
+)
+
+_operator = dict(
+    mu0=1.65,
+    _mugrid=[100.0],
+    configs=dict(
+        evolution_method="iterate-exact",
+        ev_op_max_order=[10, 0],
+        ev_op_iterations=10,
+        interpolation_polynomial_degree=4,
+        interpolation_is_log=True,
+        scvar_method=None,
+        inversion_method=None,
+        n_integration_cores=0,
+    ),
+    debug=dict(
+        skip_singlet=False,
+        skip_non_singlet=False,
+    ),
+    rotations=dict(
+        xgrid=np.geomspace(1e-7, 1.0, 50).tolist(),
```

Comment:
Maybe I would change the call to `make_lambert_grid` to `(min, max, npoints)`, to make it familiar to NumPy users (since it is the way `{lin,log,geom}space` work).

Reply:
As @felixhekhorn explained to me, `lambert_grid` is somehow different from `{lin,log,geom}space`, since it has a fixed point 1, so `max` should not be changed (other than for special debugging purpose), so it makes to have it the last one. Moreover, often you do not want even to move `min`, but just `npoints`, so the arguments are in the correct order.

The only NumPy style modification I will do is then in the function name: instead of `make_lambert_grid`, I will transition to the more Pythonic `lambertgrid` (I would have liked `lambertspace`, that would make sense, but since it is already different in the signature, let's reflect this difference also in the name).